### PR TITLE
Collect code coverage data

### PR DESCRIPTION
**Pull request by ligurio**
What has been done? Why? What problem is being solved?

I didn't forget about

- [ ] Tests
- [x] Changelog
- [ ] Documentation

Closes #???

### CODE REVIEW

**Comment by olegrok** [comment]
Is it possible to push results to coveralls?

Something like https://github.com/tarantool/graphql.0/blob/05f39946299cb2f35a97be326b992aace0205eaf/.travis.yml#L46

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 2.8 FATAL_ERROR)
 
-project(crud NONE)
+project(crud C)
```
[the remark this answers is not in the document]
Comment (ligurio) [reply CMakeLists.txt:3]: True, I was confused by compilation of tuple-merger.
This actually not needed, seems warning was triggered by `cmake/FindTarantool.cmake` that was added temporarily to a local project.

**Comment by olegrok** [comment]
Let's bump luatest version while we are here

**Comment by ligurio** [comment]
> Let's bump luatest version while we are here

bumped in a separate commit

**Review by olegrok — APPROVED** [review]
LGTM. Thanks.

**Review by AnaNek — APPROVED** [review]

**Comment by ligurio** [comment]
Moved commit "Update Tarantool versions in test matrix" to a separate PR -
https://github.com/tarantool/crud/pull/198

**Pull request merged**